Canvas/CanvasUITests/Dashboard/Dashboard.swift

```diff
@@ -48,6 +48,22 @@ enum Dashboard {
         return app.find(labelContaining: "\(percent)%")
     }
 
+    static func groupCard(id: String) -> Element {
+        return app.find(id: "group-row-\(id)")
+    }
+
+    static var dashboardTab: Element {
+        return app.find(label: "Dashboard")
+    }
+
+    static var calendarTab: Element {
+        return app.find(label: "Calendar")
+    }
+
+    static var inboxTab: Element {
+        return app.find(id: "tab-bar.inbox-btn")
+    }
```

Comment:
please use `TabBar.dashboardTab`, `TabBar.calendarTab, & `TabBar.inboxTab` rather than adding these back to `Dashboard`, since they are not a part of the dashboard, and are also visible on any one of these tabs.